src/utils/resources/shared.ts

```diff
@@ -41,20 +42,27 @@ export const getLabel = (entity: K8sResourceCommon, label: string, defaultValue?
  * function for getting a resource URL
  * @param {k8sModel} model - model to get the URL from
  * @param {K8sResourceCommon} resource - resource to get the URL from
+ * @param {string} activeNamespace - name of the actual project (the active namespace)
+
  * @returns the URL for the resource
  */
-export const getResourceUrl = (model: K8sModel, resource?: K8sResourceCommon): string => {
+export const getResourceUrl = (
+  model: K8sModel,
+  resource?: K8sResourceCommon,
+  activeNamespace?: string,
+): string => {
   if (!model) return null;
   const { crd, namespaced, plural } = model;
 
-  const namespace = resource?.metadata?.namespace
-    ? `ns/${resource.metadata.namespace}`
-    : 'all-namespaces';
+  const namespace =
+    resource?.metadata?.namespace ||
+    (activeNamespace === ALL_NAMESPACES_SESSION_KEY ? null : activeNamespace);
```

Comment:
can this also work?
const namespace =
    resource?.metadata?.namespace ||
    (activeNamespace !== ALL_NAMESPACES_SESSION_KEY && activeNamespace); ?

Reply:
Yeah, that should work, too. Just more nicely written. Thanks, Matan! 😊

Reply:
Changed!